docs/website/docs/general-usage/http/rest-client.md

```diff
@@ -446,7 +446,7 @@ The available authentication methods are defined in the `dlt.sources.helpers.res
 - [HttpBasicAuth](#http-basic-authentication)
 - [OAuth2ClientCredentials](#oauth20-authorization)
 
-For specific use cases, you can [implement custom authentication](#implementing-custom-authentication) by subclassing the `AuthBase` class from the Requests library.
+For specific use cases, you can [implement custom authentication](#implementing-custom-authentication) by subclassing the `AuthConfigBase` class from the Requests library.
```

Comment:
There's no AuthConfigBase in the Requests library, so this sentence is not correct

Reply:
Aha, so then this would be correct:
```suggestion
For specific use cases, you can [implement custom authentication](#implementing-custom-authentication) by subclassing the `AuthConfigBase` class from the `dlt.sources.helpers.rest_client.auth` module.
```

It's related to this issue: https://github.com/dlt-hub/dlt/issues/1736